packages/devtools_app/lib/src/primitives/utils.dart

```diff
@@ -242,7 +240,10 @@ Future<void> delayForBatchProcessing({int micros = 0}) async {
 /// Completes with null on timeout.
 Future<T> timeout<T>(Future<T> operation, int timeoutMillis) => Future.any<T>([
       operation,
-      Future<T>.delayed(Duration(milliseconds: timeoutMillis), () => null)
+      Future<T>.delayed(
+          Duration(milliseconds: timeoutMillis),
+          // ignore: unnecessary_parenthesis
+          (() => null) as FutureOr<T> Function()?)
```

Comment:
can we just remove the unnecessary parens?

Reply:
Analyzer gets confused by `() => null as FutureOr<T> Function()`

Reply:
@bwilkerson is this the expected behavior?